src/app/pages/datasets/modules/permissions/containers/permissions-card/permissions-card.component.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('PermissionsCardComponent', () => {
     name: 'testpool/dataset',
     mountpoint: '/mnt/testpool/dataset',
     pool: 'testpool',
-  } as Dataset;
+  } as DatasetDetails;
```

Comment:
Why test like this is affected?

Reply:
> Why test like this is affected?

This change is made so that type of test object would match the type of `PermissionsCardComponent.dataset`